src/target/adiv5.c

```diff
@@ -731,13 +732,9 @@ void adiv5_dp_init(ADIv5_DP_t *dp)
 		}
```

Comment:
this is now missing the cleanup
```
dp->designer_code = 0;
dp->partno = 0;
dp->mindp = 0;
```
realistically this doesnt make a difference, and its still unknown if this check actually catches anything